src/components/AssetSearch/components/DefaultAssetList.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Asset } from '@shapeshiftoss/types'
+import { useMemo } from 'react'
+
+import { GroupedAssetList } from './GroupedAssetList/GroupedAssetList'
+
+export type DefaultAssetListProps = {
+  isPortfolioLoading: boolean
+  isPopularAssetIdsLoading: boolean
+  portfolioAssetsSortedByBalance: Asset[]
+  popularAssets: Asset[]
+  onClickItem: (asset: Asset) => void
+}
+
+export const DefaultAssetList = ({
+  isPortfolioLoading,
```

Comment:
Any reason we drill this down?

Reply:
See https://github.com/shapeshift/web/pull/6382#discussion_r1522136278 :)

Reply:
Also more of a product question, would it make sense for this to be decoupled from the portfolio loading to make popular assets available immediately? Unless I didn't get the intent here in which case disregard me!

Reply:
@woodenfurniture q ^^